src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -241,6 +241,43 @@ def capacity(self) -> int:
         """
         return self._buffer.maxlen
 
+    def at(self, key: int | datetime) -> float:  # pylint: disable=invalid-name
```

Comment:
I actually hate this check failing because names are short. Fortunately is going away in `pylint` 3.

![image](https://github.com/frequenz-floss/frequenz-sdk-python/assets/1031485/214f932b-ec18-4fbc-bd92-40899fb447ce)

https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/invalid-name.html#predefined-naming-patterns

We could disable it now by customizing the name regexes, but I'm not sure it is worth it.